src/sources/util/tcp.rs

```diff
@@ -13,7 +13,7 @@ use futures::{
 use futures01::Sink;
 use listenfd::ListenFd;
 use serde::{de, Deserialize, Deserializer, Serialize};
-use std::{fmt, io, net::SocketAddr, task::Poll, time::Duration};
+use std::{fmt, io, mem::drop, net::SocketAddr, task::Poll, time::Duration};
```

Comment:
How `std::mem::drop` is different from available `drop`?

Reply:
Good catch, It isn't different. I'll change it in another PR.